regression/goto-instrument/every_cve/every_cve_since_2014.c

```diff
@@ -0,0 +1,82 @@
+int SSLHASHSHA1_update();
```

Comment:
May be we should call this file _named_ CVEs or _major_ CVEs since 2014? Those are the one I see here.

The total number of CVEs reported is roughly 20K per year! Many of them are low-severity ones.